### ISSUE TITLE
Changelog is not up to date

### ISSUE THREAD

**Issue by pyce-lb** [microsoft/vsts-extension-retrospectives#289]
I am always really excited when we create a new board for the retrospectives and I see little tweaks and fixes here and there or new features and options appearing and want to try them out immediately.
Unfortunately the [CHANGELOG.md](https://github.com/microsoft/vsts-extension-retrospectives/blob/master/CHANGELOG.md) which is linked to the _What's New_ button in the top right corner of the extension has not been updated for quite a while. 😢 

###### P.S. I love the extension, thank you very much for all the effort you are putting into it! :heart:

**Comment by mindlessroman** [comment]
You're right! The changelog should probably be updated! @polatengin - this might be good ahead of the next release!
